src/pages/get-started/create-integration.md

```diff
@@ -0,0 +1,332 @@
+
+In the following example, the test integration requires access to the following resources in the Sales module:
+
+```xml
+<integrations>
+    <integration name="TestIntegration">
+        <resources>
+            <!-- To grant permission to Magento_Log::online, its parent Magento_Customer::customer needs to be declared as well-->
+            <resource name="Magento_Customer::customer" />
+            <resource name="Magento_Log::online" />
+            <!-- To grant permission to Magento_Sales::reorder, all its parent resources need to be declared-->	
+            <resource name="Magento_Sales::sales" />
+            <resource name="Magento_Sales::sales_operation" />
+            <resource name="Magento_Sales::sales_order" />
+            <resource name="Magento_Sales::actions" />
+            <resource name="Magento_Sales::reorder" />
+        </resources>
+    </integration>
+</integrations>
+```
+
+### Pre-configure the integration
+
+Your module can optionally provide values in configuration file `config.xml`, so that the integration can be automatically pre-configured with default values. To enable this feature, update the `config.xml` file in the `etc/integration` directory.
+
+<InlineAlert variant="info" slots="text"/>
+
+If you pre-configure the integration, the values cannot be edited from the [admin](https://glossary.magento.com/admin) panel.
+
+The file defines which API resources the integration has access to.
+
+```xml
+<integrations>
+   <integration name="TestIntegration">
+       <email></email>
+       <endpoint_url></endpoint_url>
+       <identity_link_url></identity_link_url>
+   </integration>
+</integrations>
+```
+
+<table>
+<tr>
+<th>Element</th>
+<th>Description</th>
+</tr>
+<tr>
+<td>integrations</td>
+<td>Contains one or more integration definitions.</td>
+</tr>
+<tr>
+<td>integration name=""</td>
+<td>Defines an integration. The <inlineCode class="spectrum-Body--sizeS">name</inlineCode> must be specified.</td>
+</tr>
+<tr>
+<td>email</td>
+<td>An email to associate with this integration.</td>
+</tr>
+<tr>
+<td>endpoint_url</td>
+<td><p>Optional. The [URL](https://glossary.magento.com/url) where OAuth credentials can be sent when using OAuth for token exchange. We strongly recommend using <inlineCode class="spectrum-Body--sizeS">https://</inlineCode>.</p>
+<p>See <a href="./authentication/gs-authentication-oauth">OAuth-based authentication</a> for details.</p></td>
+</tr>
+<tr>
+<td>identity_link_url</td>
+<td>Optional. The URL that redirects the user to link their 3rd party account with the Commerce integration.</td>
+</tr>
+</table>
+
+## Install your module
+
+Use the following steps to install your module:
+
+1. Run the following command to update the [database schema](https://glossary.magento.com/database-schema) and data.
+
+   ```bash
+   bin/magento setup:upgrade
+   ```
+
+1. Run the following command to generate the new code.
+
+    <InlineAlert variant="info" slots="text"/>
```

Comment:
This is another multiparagraph note that doesn't render. I would create an unofficial note like this: **Note:** In Production mode...

Reply:
it seems we can use `<div>` tags to include multiple paragraphs.